build.gradle

```diff
@@ -70,8 +70,8 @@ ext {
     androidCompileSdkVersion = 24
     androidBuildToolsVersion = '25.0.0'
     androidSupportSdkVersion = '25.0.0'
-    minSdkVersion = 16
-    targetSdkVersion = 24
+    minSdkVersion = 14
+    targetSdkVersion = 25
```

Comment:
I keep `targetSdkVersion` at 24 because the source code of 25 is not available, AndroidStudio could not use 24's source code, could you please change it back to 24?